docs/central-users.rst

```diff
@@ -19,67 +19,94 @@ Central features Role-based User permissioning. In the current release of Centra
+| View & Download                  | x             | x               | x              |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| OData Access                     | x             | x               | x              |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| **Project Entity Lists**                                                                             |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Create                           | x             | x               |                |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| List All                         | x             | x               | x              |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Add Properties                   | x             | x               |                |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Update Settings                  | x             | x               |                |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| **Project Entities**                                                                                 |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Create from Submission           | x             | x               |                | x              |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Create Directly                  | x             | x               |                |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| View in Linked Forms             | x             | x               | x              | x              |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| | View & Download Anywhere,      | x             | x               | x              |                |
+| | Not Just in Linked Forms       |               |                 |                |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| OData Access                     | x             | x               | x              |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Update Data                      | x             | x               |                |                |
```

Comment:
We use the term "edit" above for submissions, but in Frontend, we only talk about "updating" entities.